### PR TITLE
Fixing Broken Latex Code

### PR DESCRIPTION
**Pull request by bartnikm**
**Description**

Places were latex code did not display correctly were updated to allow the code to display correctly. 

**Motivation and context**

There were some places where code written in latex was formatted or written incorrectly

**How has this been tested?**
- [ ] Testing pipeline.
- [ ] Other. 

**Examples**

https://bartnikm.github.io/tardis-bartnikm/branch/latex-names/physics/setup/plasma/helium_nlte.html

**Type of change**

- [ ] Bug fix. 
- [ ] New feature. 
- [ ] Breaking change. 
- [ ] None of the above. 

**Checklist**

- [x] My change requires a change to the documentation.
    - [x] I have updated the documentation accordingly.
    - [x] (optional) I have built the documentation on my fork following [the instructions](https://tardis-sn.github.io/tardis/contributing/development/documentation_guidelines.html#sharing-the-built-documentation-in-your-pr-documentation-preview).
- [ ] I have assigned and requested two reviewers for this pull request.

### CODE REVIEW

File: docs/physics/setup/plasma/helium_nlte.rst [hunk 1]
```diff
@@ -48,4 +48,4 @@ In the TARDIS plasma, some of these equations are re-written slightly to make us
 Numerical He NLTE
 ------------------
 
-Another `helium_treatment` option offered by TARDIS is `numerical-nlte`. The use of this plasma property requires an additional code that is the property of Stephan Hachinger (see arXiv:1201.1506) and is not distributed with TARDIS. TARDIS also requires a specific atomic datafile to use this module. This plasma option is included so that people who have access to and permission to use the necessary module may use it. Otherwise, the `recomb-NLTE` option provides a very accurate alternative approximation.
\ No newline at end of file
+Another `helium_treatment` option offered by TARDIS is `numerical-nlte`. The use of this plasma property requires an additional code that is the property of Stephan Hachinger (see arXiv:1201.1506) and is not distributed with TARDIS. TARDIS also requires a specific atomic datafile to use this module. This plasma option is included so that people who have access to and permission to use the necessary module may use it. Otherwise, the `recomb-NLTE` option provides a very accurate alternative approximation.
```
Comment (andrewfullard) [inline docs/physics/setup/plasma/helium_nlte.rst:51]: What actually changed here? I can't tell from the changelog

Comment (bartnikm) [reply docs/physics/setup/plasma/helium_nlte.rst:51]: I did not edit or look at this section, so I'm not sure what, if anything, was changed here either.

Comment (smithis7) [reply docs/physics/setup/plasma/helium_nlte.rst:51]: I have found that sometimes github just does that

**Comment by smithis7** [comment]
I think we should wait on this one until #1868 has been merged. That way we can rebase and check that this does fix the issue in the plasma graph. But it looks good to me

File: docs/physics/setup/plasma/helium_nlte.rst (hunk 1 above)
Comment (wkerzendorf) [reply docs/physics/setup/plasma/helium_nlte.rst:51]: normally github does not change things. What does your local diff say?

Comment (bartnikm) [reply docs/physics/setup/plasma/helium_nlte.rst:51]: this is potentially a different newline character due to this being edited via Jupyter Notebook on a Windows OS.

**Review by wkerzendorf — APPROVED** [review]

**Review by andrewfullard — APPROVED** [review]

**Pull request merged**